cpp/include/cudf/quantiles.hpp

```diff
@@ -42,5 +42,36 @@ quantile(column_view const& input,
          interpolation interp = interpolation::LINEAR,
          order_info column_order = {});
 
+/* @brief Gathers the rows corrosponding to the desired quantiles.
+ *
+ * The indices used to gather rows are computed by interpolating between the
```

Comment:
Since I often have to look it up, I think it would be good to define quantile here.
```suggestion
 * Quantiles are cut points that divide the range of a dataset into continuous intervals. e.g. quartiles
 * are the three cut points that divide a dataset into four equal-sized groups. See 
 * https://en.wikipedia.org/wiki/Quantile
 *
 * The indices used to gather rows are computed by interpolating between the
```